src/Constants.ts

```diff
@@ -36,3 +36,5 @@ export const Permissions = Object.freeze({
   CREATE_INSTANT_INVITE: BigFlagUtils.getFlag(0),
   ADMINISTRATOR: BigFlagUtils.getFlag(3),
 });
+
+export const HANDSHAKE_SDK_VERSION_MINIUM_MOBILE_VERSION = '250.5';
```

Comment:
Dumb question, is it okay to use this string for both iOS and Android? I remember sometimes one had a `<version>.0` and the other is just `<version>`